actions/createIdentity.js

```diff
@@ -1,23 +0,0 @@
-import {
-  CALL_API,
-  NEW_IDENTITY_REQUEST,
-  NEW_IDENTITY_FAILURE,
-  NEW_IDENTITY_SUCCESS,
```

Comment:
These constants no longer to use except `CALL_API `, it would be great if we could remove it from `constants.js` and `reducer/users.js`. 😃

Reply:
But we still need the constants for state checking, right ?

Reply:
Since `createIdentity` action is removed, the action type will not be matched with the following constants:
 - `NEW_IDENTITY_REQUEST `
 - `NEW_IDENTITY_FAILURE `
 - `NEW_IDENTITY_SUCCESS `.

It is okay to remove these constants for state checking.

Reply:
Ok. I'll add another commit to remove it.